server/service/integration_enterprise_test.go

```diff
@@ -64,6 +65,9 @@ func (s *integrationEnterpriseTestSuite) SetupSuite() {
 		Logger:         log.NewLogfmtLogger(os.Stdout),
 		EnableCachedDS: true,
 	}
+	if os.Getenv("FLEET_INTEGRATION_TESTS_DISABLE_LOG") != "" {
+		config.Logger = kitlog.NewNopLogger()
```

Comment:
How do we feel about this approach and the env var name? If we agree with it, I'll push a subsequent PR with contributor docs to mention it. With all logs printed out, it can be a lot of trouble finding which test failed when running the suite locally.

Reply:
sounds good to me! 👍

Reply:
https://github.com/fleetdm/fleet/pull/17524